Source/WebKit/UIProcess/WebPageProxyInternals.h

```diff
@@ -215,6 +215,8 @@ struct WebPageProxy::Internals final : WebPopupMenuProxy::Client
     PageAllowedToRunInTheBackgroundCounter::Token pageAllowedToRunInTheBackgroundToken;
 
     HashMap<WebCore::RegistrableDomain, WeakPtr<RemotePageProxy>> domainToRemotePageProxyMap;
+    RefPtr<RemotePageProxy> remotePageProxyInOpenerProcess;
+    HashSet<Ref<RemotePageProxy>> openedRemotePageProxies;
```

Comment:
seems like we're leaking these objects here?

Reply:
There's a FIXME comment by WebPageProxy::addOpenedRemotePageProxy